crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -107,13 +119,27 @@ pub(crate) fn create(
                 {
                     debug!("Ignoring empty directory");
                 } else {
-                    return Err(Error::Io(io::Error::new(
-                        io::ErrorKind::AlreadyExists,
-                        format!(
-                            "The directory `{}` exists, but it's not a virtual environment",
-                            location.user_display()
-                        ),
-                    )));
+                    if Term::stderr().is_term() {
+                        return Err(Error::Io(io::Error::new(
```

Comment:
It's a little weird that this is separated from the confirmation prompt, it makes it hard to tell what the logic is. Could it be restructured to be clearer?

Reply:
Addressed by restructuring